source/Contribution-Guide/guide-examples/how-to-guide.html.md.erb

```diff
@@ -0,0 +1,33 @@
+---
+title: How-To Guide Template
+last_reviewed_on: 2025-02-25
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+## How to [Perform Specific Task] in [System/Application Name]
+This guide provides step-by-step instructions for [performing specific task] in [System/Application Name].
+
+## Prerequisites
+- Access level required for this task
+- Any necessary tools or software
+- Required background knowledge
+
+## How-to steps
```

Comment:
Do we want to specifically call out the use of pictures and example commands/PRs?

Reply:
added